roles/scalelite_server/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Get BBB servers YAML
+  community.docker.docker_container_exec:
+    container: scalelite-api
+    command: ./bin/rake servers:yaml[verbose]
+  register: scalelite_bbb_servers_yaml
+  changed_when: false
+  check_mode: false
+
+- name: Filter server list
+  set_fact:
+    bbb_server_list: "{{ (scalelite_bbb_servers_yaml.stdout | from_yaml)[':servers']
+      | dict2items | selectattr('value.:url', 'equalto', 'https://' + bbb_server_name + '/bigbluebutton/api') }}"
+
+- name: Check if server is present
+  set_fact:
+    bbb_server_present: "{{ bbb_server_list | length > 0 }}"
+
+- name: Register server
+  community.docker.docker_container_exec:
+    container: scalelite-api
+    command: "./bin/rake servers:add[https://{{ bbb_server_name }}/bigbluebutton/api,{{ bbb_api_key }}]"
+  register: bbb_server_id
+  when: not ansible_check_mode and not bbb_server_present and bbb_server_state == 'present'
+
+- name: Enable server
+  community.docker.docker_container_exec:
+    container: scalelite-api
+    command: "./bin/rake servers:enable[{{ (bbb_server_id.stdout_lines[1] | from_yaml).id }}]"
+  ignore_errors: '{{ ansible_check_mode }}'
+  register: bbb_server_id
+  when: not ansible_check_mode and not bbb_server_present and bbb_server_state == 'present'
+
+- name: Delete server
+  community.docker.docker_container_exec:
+    container: scalelite-api
+    command: "./bin/rake servers:remove[{{ bbb_server_list[0].key }}]"
+  ignore_errors: '{{ ansible_check_mode }}'
+  when: not ansible_check_mode and bbb_server_present and bbb_server_state == 'absent'
```

Comment:
Is the `ignore_errors: '{{ ansible_check_mode }}'` necessary if the task is not executed in this case? (`when: not ansible_check_mode`)

Reply:
The intention was to have Ansible indicate if the server was added or if it was already there when running in check mode, but it won't work like that, I'll fix it.